diff --git a/node_modules/scrypt/build/Makefile b/node_modules/scrypt/build/Makefile
index 7f2bedb..548dfbc 100644
--- a/node_modules/scrypt/build/Makefile
+++ b/node_modules/scrypt/build/Makefile
@@ -323,8 +323,8 @@ ifeq ($(strip $(foreach prefix,$(NO_LOAD),\
 endif
 
 quiet_cmd_regen_makefile = ACTION Regenerating $@
-cmd_regen_makefile = cd $(srcdir); /home/noah/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py -fmake --ignore-environment "--toplevel-dir=." -I/tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/build/config.gypi -I/home/noah/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/node-gyp/addon.gypi -I/home/noah/.node-gyp/10.13.0/include/node/common.gypi "--depth=." "-Goutput_dir=." "--generator-output=build" "-Dlibrary=shared_library" "-Dvisibility=default" "-Dnode_root_dir=/home/noah/.node-gyp/10.13.0" "-Dnode_gyp_dir=/home/noah/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/node-gyp" "-Dnode_lib_file=/home/noah/.node-gyp/10.13.0/<(target_arch)/node.lib" "-Dmodule_root_dir=/tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt" "-Dnode_engine=v8" binding.gyp
-Makefile: $(srcdir)/../../../../home/noah/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/node-gyp/addon.gypi $(srcdir)/build/config.gypi $(srcdir)/binding.gyp $(srcdir)/../../../../home/noah/.node-gyp/10.13.0/include/node/common.gypi
+cmd_regen_makefile = cd $(srcdir); /home/noah/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/node-gyp/gyp/gyp_main.py -fmake --ignore-environment "--toplevel-dir=." -I/home/noah/github/renproject/renvm.js/node_modules/scrypt/build/config.gypi -I/home/noah/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/node-gyp/addon.gypi -I/home/noah/.node-gyp/10.13.0/include/node/common.gypi "--depth=." "-Goutput_dir=." "--generator-output=build" "-Dlibrary=shared_library" "-Dvisibility=default" "-Dnode_root_dir=/home/noah/.node-gyp/10.13.0" "-Dnode_gyp_dir=/home/noah/.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/node-gyp" "-Dnode_lib_file=/home/noah/.node-gyp/10.13.0/<(target_arch)/node.lib" "-Dmodule_root_dir=/home/noah/github/renproject/renvm.js/node_modules/scrypt" "-Dnode_engine=v8" binding.gyp
+Makefile: $(srcdir)/../../../../../.node-gyp/10.13.0/include/node/common.gypi $(srcdir)/build/config.gypi $(srcdir)/binding.gyp $(srcdir)/../../../../../.nvm/versions/node/v10.13.0/lib/node_modules/npm/node_modules/node-gyp/addon.gypi
 	$(call do_cmd,regen_makefile)
 
 # "all" is a concatenation of the "all" targets from all the included
diff --git a/node_modules/scrypt/build/binding.Makefile b/node_modules/scrypt/build/binding.Makefile
index f7497eb..8c7d3e9 100644
--- a/node_modules/scrypt/build/binding.Makefile
+++ b/node_modules/scrypt/build/binding.Makefile
@@ -3,4 +3,4 @@
 export builddir_name ?= ./build/.
 .PHONY: all
 all:
-	$(MAKE) copied_files scrypt_lib scrypt_wrapper scrypt
+	$(MAKE) scrypt_wrapper copied_files scrypt_lib scrypt
diff --git a/node_modules/scrypt/build/config.gypi b/node_modules/scrypt/build/config.gypi
index 22f10c7..d410103 100644
--- a/node_modules/scrypt/build/config.gypi
+++ b/node_modules/scrypt/build/config.gypi
@@ -73,7 +73,6 @@
     "version_commit_hooks": "true",
     "user_agent": "yarn/1.17.0 npm/? node/v10.13.0 linux x64",
     "bin_links": "true",
-    "wrap_output": "",
     "init_version": "1.0.0",
     "init_license": "MIT",
     "version_tag_prefix": "v",
diff --git a/node_modules/scrypt/index.js b/node_modules/scrypt/index.js
index 7b13a79..348c657 100644
--- a/node_modules/scrypt/index.js
+++ b/node_modules/scrypt/index.js
@@ -1,6 +1,6 @@
 "use strict";
 
-var scryptNative = require("./build/Release/scrypt")
+var scryptNative = require("scrypt")
   , Crypto = require("crypto")
   , Os = require("os");
 
diff --git a/node_modules/scrypt/scrypt/scrypt-1.2.0/Makefile b/node_modules/scrypt/scrypt/scrypt-1.2.0/Makefile
index 31efe7b..1315a22 100644
--- a/node_modules/scrypt/scrypt/scrypt-1.2.0/Makefile
+++ b/node_modules/scrypt/scrypt/scrypt-1.2.0/Makefile
@@ -257,12 +257,12 @@ distuninstallcheck_listfiles = find . -type f -print
 am__distuninstallcheck_listfiles = $(distuninstallcheck_listfiles) \
   | sed 's|^\./|$(prefix)/|' | grep -v '$(infodir)/dir$$'
 distcleancheck_listfiles = find . -type f -print
-ACLOCAL = ${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing aclocal-1.15
+ACLOCAL = ${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing aclocal-1.15
 AMTAR = $${TAR-tar}
 AM_DEFAULT_VERBOSITY = 1
-AUTOCONF = ${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing autoconf
-AUTOHEADER = ${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing autoheader
-AUTOMAKE = ${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing automake-1.15
+AUTOCONF = ${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing autoconf
+AUTOHEADER = ${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing autoheader
+AUTOMAKE = ${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing automake-1.15
 AWK = mawk
 CC = gcc
 CCDEPMODE = depmode=gcc3
@@ -288,7 +288,7 @@ LIBOBJS =
 LIBS = -lrt 
 LTLIBOBJS = 
 MAINT = #
-MAKEINFO = ${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing makeinfo
+MAKEINFO = ${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing makeinfo
 MKDIR_P = /bin/mkdir -p
 OBJEXT = o
 PACKAGE = scrypt
@@ -304,10 +304,10 @@ SET_MAKE =
 SHELL = /bin/bash
 STRIP = 
 VERSION = 1.2.0
-abs_builddir = /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0
-abs_srcdir = /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0
-abs_top_builddir = /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0
-abs_top_srcdir = /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0
+abs_builddir = /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0
+abs_srcdir = /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0
+abs_top_builddir = /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0
+abs_top_srcdir = /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0
 ac_ct_CC = gcc
 am__include = include
 am__leading_dot = .
@@ -326,7 +326,7 @@ host_alias =
 htmldir = ${docdir}
 includedir = ${prefix}/include
 infodir = ${datarootdir}/info
-install_sh = ${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/install-sh
+install_sh = ${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/install-sh
 libdir = ${exec_prefix}/lib
 libexecdir = ${exec_prefix}/libexec
 localedir = ${datarootdir}/locale
diff --git a/node_modules/scrypt/scrypt/scrypt-1.2.0/config.log b/node_modules/scrypt/scrypt/scrypt-1.2.0/config.log
index c2e165a..100908c 100644
--- a/node_modules/scrypt/scrypt/scrypt-1.2.0/config.log
+++ b/node_modules/scrypt/scrypt/scrypt-1.2.0/config.log
@@ -27,14 +27,8 @@ uname -v = #26-Ubuntu SMP Mon Jun 24 09:32:08 UTC 2019
 /usr/bin/oslevel       = unknown
 /bin/universe          = unknown
 
-PATH: /tmp/yarn--1564057731880-0.1244035174293967
-PATH: /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/node_modules/.bin
-PATH: /home/noah/.config/yarn/link/node_modules/.bin
-PATH: /home/noah/.nvm/versions/node/v10.13.0/libexec/lib/node_modules/npm/bin/node-gyp-bin
-PATH: /home/noah/.nvm/versions/node/v10.13.0/lib/node_modules/npm/bin/node-gyp-bin
-PATH: /home/noah/.nvm/versions/node/v10.13.0/bin/node_modules/npm/bin/node-gyp-bin
-PATH: /tmp/yarn--1564057728607-0.9575453125802653
-PATH: /home/noah/github/renproject/renvm.js/node_modules/.bin
+PATH: /tmp/yarn--1564057247369-0.6001871984664848
+PATH: /home/noah/github/renproject/renvm.js/node_modules/scrypt/node_modules/.bin
 PATH: /home/noah/.config/yarn/link/node_modules/.bin
 PATH: /home/noah/.nvm/versions/node/v10.13.0/libexec/lib/node_modules/npm/bin/node-gyp-bin
 PATH: /home/noah/.nvm/versions/node/v10.13.0/lib/node_modules/npm/bin/node-gyp-bin
@@ -465,7 +459,7 @@ am_cv_prog_cc_c_o=yes
 ## Output variables. ##
 ## ----------------- ##
 
-ACLOCAL='${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing aclocal-1.15'
+ACLOCAL='${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing aclocal-1.15'
 AMDEPBACKSLASH='\'
 AMDEP_FALSE='#'
 AMDEP_TRUE=''
@@ -474,9 +468,9 @@ AM_BACKSLASH='\'
 AM_DEFAULT_V='$(AM_DEFAULT_VERBOSITY)'
 AM_DEFAULT_VERBOSITY='1'
 AM_V='$(V)'
-AUTOCONF='${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing autoconf'
-AUTOHEADER='${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing autoheader'
-AUTOMAKE='${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing automake-1.15'
+AUTOCONF='${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing autoconf'
+AUTOHEADER='${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing autoheader'
+AUTOMAKE='${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing automake-1.15'
 AWK='mawk'
 CC='gcc'
 CCDEPMODE='depmode=gcc3'
@@ -503,7 +497,7 @@ LTLIBOBJS=''
 MAINT='#'
 MAINTAINER_MODE_FALSE=''
 MAINTAINER_MODE_TRUE='#'
-MAKEINFO='${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing makeinfo'
+MAKEINFO='${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing makeinfo'
 MKDIR_P='/bin/mkdir -p'
 OBJEXT='o'
 PACKAGE='scrypt'
@@ -542,7 +536,7 @@ host_alias=''
 htmldir='${docdir}'
 includedir='${prefix}/include'
 infodir='${datarootdir}/info'
-install_sh='${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/install-sh'
+install_sh='${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/install-sh'
 libdir='${exec_prefix}/lib'
 libexecdir='${exec_prefix}/libexec'
 localedir='${datarootdir}/locale'
diff --git a/node_modules/scrypt/scrypt/scrypt-1.2.0/config.status b/node_modules/scrypt/scrypt/scrypt-1.2.0/config.status
index d66c7e5..1138e0a 100755
--- a/node_modules/scrypt/scrypt/scrypt-1.2.0/config.status
+++ b/node_modules/scrypt/scrypt/scrypt-1.2.0/config.status
@@ -437,7 +437,7 @@ Copyright (C) 2012 Free Software Foundation, Inc.
 This config.status script is free software; the Free Software Foundation
 gives unlimited permission to copy, distribute and modify it."
 
-ac_pwd='/tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0'
+ac_pwd='/home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0'
 srcdir='.'
 INSTALL='/usr/bin/install -c'
 MKDIR_P='/bin/mkdir -p'
@@ -656,12 +656,12 @@ S["mkdir_p"]="$(MKDIR_P)"
 S["MKDIR_P"]="/bin/mkdir -p"
 S["INSTALL_STRIP_PROGRAM"]="$(install_sh) -c -s"
 S["STRIP"]=""
-S["install_sh"]="${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/install-sh"
-S["MAKEINFO"]="${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing makeinfo"
-S["AUTOHEADER"]="${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing autoheader"
-S["AUTOMAKE"]="${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing automake-1.15"
-S["AUTOCONF"]="${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing autoconf"
-S["ACLOCAL"]="${SHELL} /tmp/tmp-13800h8lNngnR5Ppo/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing aclocal-1.15"
+S["install_sh"]="${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/install-sh"
+S["MAKEINFO"]="${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing makeinfo"
+S["AUTOHEADER"]="${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing autoheader"
+S["AUTOMAKE"]="${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing automake-1.15"
+S["AUTOCONF"]="${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing autoconf"
+S["ACLOCAL"]="${SHELL} /home/noah/github/renproject/renvm.js/node_modules/scrypt/scrypt/scrypt-1.2.0/config.aux/missing aclocal-1.15"
 S["VERSION"]="1.2.0"
 S["PACKAGE"]="scrypt"
 S["CYGPATH_W"]="echo"
