diff --git a/node_modules/web3-utils/types/index.d.ts b/node_modules/web3-utils/types/index.d.ts
index d3a1104..ba73daa 100644
--- a/node_modules/web3-utils/types/index.d.ts
+++ b/node_modules/web3-utils/types/index.d.ts
@@ -21,7 +21,7 @@
  */
 
 import BN = require('bn.js');
-import {Buffer} from 'buffer';
+import { Buffer } from "buffer";
 
 export type Unit =
     | 'noether'
@@ -57,13 +57,13 @@ export type Mixed =
     | number
     | BN
     | {
-          type: string;
-          value: string;
-      }
+        type: string;
+        value: string;
+    }
     | {
-          t: string;
-          v: string | BN | number;
-      }
+        t: string;
+        v: string | BN | number;
+    }
     | boolean;
 
 export type Hex = string | number;
@@ -114,7 +114,7 @@ export function getUnitValue(unit: Unit): string;
 export function unitMap(): Units;
 export function testAddress(bloom: string, address: string): boolean;
 export function testTopic(bloom: string, topic: string): boolean;
-export function getSignatureParameters(signature: string): {r: string; s: string; v: number};
+export function getSignatureParameters(signature: string): { r: string; s: string; v: number };
 export function stripHexPrefix(str: string): string;
 
 // interfaces
@@ -164,7 +164,7 @@ export interface Utils {
     unitMap(): Units;
     testAddress(bloom: string, address: string): boolean;
     testTopic(bloom: string, topic: string): boolean;
-    getSignatureParameters(signature: string): {r: string; s: string; v: number};
+    getSignatureParameters(signature: string): { r: string; s: string; v: number };
     stripHexPrefix(str: string): string;
 }
 
@@ -216,11 +216,13 @@ export interface AbiInput {
     name: string;
     type: string;
     indexed?: boolean;
-	components?: AbiInput[];
+    components?: AbiInput[];
+    internalType?: string;
 }
 
 export interface AbiOutput {
     name: string;
     type: string;
-	components?: AbiOutput[];
+    components?: AbiOutput[];
+    internalType?: string;
 }
